tools/pipeline2dot.py

```diff
@@ -65,10 +62,8 @@ def get_process_details(pid):
 
     data = {'user':'', 'cpu':0.0, 'mem':0.0, 'etime':'00:00', 'threads':0}
     try:
-        output = subprocess.check_output('ps o user,pcpu,pmem,etime,nlwp %i' % pid, shell=True)
-        if sys.version_info.major > 2 and isinstance(output, bytes):
-            # decode the output to utf-8 in python 3
-            output = output.decode("utf-8")
+        output = subprocess.check_output(f"ps o user,pcpu,pmem,etime,nlwp {pid}", shell=True)
```

Comment:
This would be another place to think about if we need `shell=True` or not.

Reply:
See [f5b0b02](https://github.com/ledatelescope/bifrost/pull/216/commits/f5b0b02cb1cc8cd446a3f4d90cfc824276c56b64).